imagetagger/imagetagger/images/templates/images/imageset.html

```diff
@@ -193,7 +193,7 @@ <h2 class="panel-title pull-left" style="padding-top: 5px;">{{imageset.name}} {{
                                 </tr>
                                 <tr>
                                     <td>Description</td>
-                                    <td>{{imageset.description}}</td>
+                                    <td>{{imageset_description | safe}}</td>
```

Comment:
Do not do that: instead, mark the string save using `mark_safe` (cf. [the docs](https://docs.djangoproject.com/en/2.1/ref/utils/#django.utils.safestring.mark_safe).)
This way, you do not need the safety (by the way, "safety" is not the term you want - "security" would be it) warnings: as you do not mark the string safe in the template, it is not inserted as HTML if it was not explicitly marked safe before (and you only mark it safe after cleaning it).